hydra-node/src/Hydra/Options.hs

```diff
@@ -285,3 +414,95 @@ parseHydraOptions = getArgs <&> parseHydraOptionsFromString >>= handleParseResul
 -- | Pure parsing of `Option` from a list of arguments.
 parseHydraOptionsFromString :: [String] -> ParserResult Options
 parseHydraOptionsFromString = execParserPure defaultPrefs hydraNodeOptions
+
+-- | Convert an 'Options' instance into the corresponding list of command-line arguments.
+--
+-- This is useful in situations where one wants to programatically define 'Options', providing
+-- some measure of type safety, without having to juggle with strings.
+toArgs :: Options -> [String]
```

Comment:
Could: name this `renderHydraOptions` or so as it resembles the inverse to `parseHydraOptionsFromString` just above (assuming `render` is a good inverse to `parse`)